static/app/views/settings/account/accountNotificationFineTuning.tsx

```diff
@@ -178,7 +165,18 @@ export default class AccountNotificationFineTuning extends AsyncView<Props, Stat
   }
 
   renderBody() {
-    const {fineTuneType} = this.props.params;
+    const {params, organizations} = this.props;
+    const {fineTuneType} = params;
+
+    if (
+      ['alerts', 'deploy', 'workflow'].includes(fineTuneType) &&
+      organizations.some(organization =>
```

Comment:
If a user is in 2 orgs, one with and one without the feature and they set their global delivery method to Slack, won't we drop notifications for the org without the feature?

Reply:
The user would not be able to set an `ExternalActor` for the second org and would get zero notifications.